.github/workflows/nightly.yml

```diff
@@ -1,10 +1,12 @@
-name: TypeScript's Nightly
-
+name: TypeScript's nightly
 on:
   workflow_dispatch:
   schedule:
     - cron: "0 12 * * *"
 
+permissions:
+  issues: write
+
```

Comment:
I think my preference would be to follow the semantics we have for our other workflows, and set the global permissions to none:

```
permissions: {}
```

and apply the required permissions to the jobs that need them.

Obviously here we only have one job, but as a best practice I feel we should keep it consistent.

Also pinging @fubhy here for additional review.

Reply:
Agreed. I simply didn’t think deep enough. The code was failing while I was testing it and these two lines made it work. Hence in this context they looked like a solution, not like a problem at all.

Reply:
Thanks for the insight. I have just pushed an update. Before doing that I also updated the example repo to be sure that all work as expected.